lib/gantree/release_notes.rb

```diff
@@ -78,11 +78,15 @@ def execute(cmd)
     end
 
     def notes
-      compare = "#{previous_sha}...#{current_sha}"
-      notes = <<-EOL
+      if previous_sha
+        compare = "#{previous_sha}...#{current_sha}"
```

Comment:
@feelobot @sillypog thanks guys. saw this but just getting to this now because we just ran into it again :)  fixing it by just making sure previous_sha1 is always set